golem/task/requestedtaskmanager.py

```diff
@@ -362,6 +362,8 @@ def get_subtask_outputs_dir(self, task_id: TaskId) -> Path:
             logger.info(
                 'app_client created for app_id=%r, clients=%r',
                 app_id, self._app_clients[app_id])
+            import time
+            time.sleep(2)
```

Comment:
Also makes me sad, maybe we can at least use asyncio?

```suggestion
            await asyncio.sleep(2)
```

Reply:
Maybe let's just move it to the Task API repo since it's an issue there.

Reply:
👍

Reply:
Server health check may help here https://github.com/golemfactory/task-api/pull/36

Reply:
that will solve it :)